src/bot/eval.py

```diff
@@ -1,25 +1,30 @@
+"""
+This module contains functions
+for evaluating code using the Coliru API.
+"""
+
 import datetime
 import discord
 
 from discord.ext import commands
-from util import coliru
+from util import coliru  # pylint: disable=no-name-in-module
```

Comment:
> These files are autogenerated and we didn't use them anywhere yet, but as far as I know they are required for scrapy to be able to correctly run the project (which is also why they're included)

Can you elaborate on this? I don't understand. With `value`, do you mean the module? Since Python doesn't raise an error, it exists?